catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -243,48 +243,60 @@ public ReliableResourceStatus call() {
 
                 bytesRead.addAndGet(n);
 
-                if (cacheFileOutputStream != null) {
```

Comment:
I don't really agree. Since those are private methods, whoever changes those in the future needs to be aware of the implementation details, including potential synchronization issues. Also, if you apply that logic, it pretty much means that you can never extract private methods that are inside synchronized block, which makes it difficult to keep the code clean. I'd rather add comments on those private methods rather than leaving the code the way it was.